pages/style-guide/html.md

```diff
@@ -4,74 +4,170 @@
 
```

Comment:
While extending this page it might be better to add a TOC similar to the JavaScript style guide? (possible by adding `"toc": true`).

Would it be a good idea here too?